configs/bundle_config.yaml

```diff
@@ -336,3 +336,16 @@ databundles:
   #   output: [cutouts/cutout-2013-era5.nc]
   #   disable_by_opt:
   #     build_cutout: [all]
+
+  # # Adding a cutout for South America, approx 18 GB
+  databundles:
+  bundle_cutouts_southamerica:
```

Comment:
My feeling is that it can be enough to remove `databundles:` on line 341, then tabs would be alright as it's now. 

Actually, we still have `databundles` in the very beginning of `bundle_config.yaml`. I suspect that duplicating it there may by a reason on the current CI complains.

Reply:
Katia, you are alright XD a good night sleep may have helped XD
CI is rerunning, as it merges it is ready to go :)

Reply:
@davide-f We appreciate your support and commitment :) Although, agree that good sleep makes wonders ;)